cmake/Utils.cmake

```diff
@@ -32,7 +32,7 @@ FUNCTION(ILINK from to)
   get_filename_component(LNTDIR ${to} DIRECTORY)
   get_filename_component(LNTNAME ${from} NAME)
   file(RELATIVE_PATH LNLNK ${LNTDIR} ${from})
-  install(CODE "EXECUTE_PROCESS(COMMAND ln -sf ${LNLNK} ./${LNTNAME}
+  install(CODE "EXECUTE_PROCESS(COMMAND ln -sf ./${LNLNK} ./${LNTNAME}
```

Comment:
Why is this?

Reply:
This is because ${LNLNK} turns out to be something like `swipl/lib/<arch>...` if we are linking under the same parent directory; and of course the symbolic link is broken then, since we need `./swipl/lib/<arch>`.

Reply:
hmmm you're right, confused it with another error.